src/js/components/MaskedInput/doc.js

```diff
@@ -50,6 +50,16 @@ export const doc = MaskedInput => {
 };
 
 export const themeDoc = {
+  'maskedInput.extend': {
+    description: 'Any additional style for MaskedInput.',
+    type: 'string | (props) => {}',
+    defaultValue: undefined,
+  },
+  'text.medium': {
+    description: 'The size of the text for the maskedInput.',
```

Comment:
`description: 'The size of the text for MaskedInput.'` to be consistent with `maskedInput.extend`